front_end/panels/ai_chat/core/EnvironmentConfig.ts

```
@@ -0,0 +1,336 @@
+  GROQ_API_KEY?: string;
+  LITELLM_API_KEY?: string;
+  timestamp?: string;
+  source?: string;
+}
+
+// Get runtime configuration if available (from Docker runtime injection)
+function getRuntimeConfig(): RuntimeConfig | null {
+  if (typeof window === 'undefined') {
+    return null;
+  }
+  
```

ESLint

[error] Trailing spaces not allowed.

(@stylistic/no-trailing-spaces)